### PR TITLE
add changelog for upcoming major version

### PR DESCRIPTION
**Pull request by dirk-thomas**
In preparation of releasing a new major version of this package.

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -2,6 +2,50 @@
...
+
+* New features
+
+  * support colcon build tool using a configuration option (`#585 <https://github.com/ros-infrastructure/ros_buildfarm/pull/585>`_, `#589 <https://github.com/ros-infrastructure/ros_buildfarm/pull/589>`_, `#591 <https://github.com/ros-infrastructure/ros_buildfarm/pull/591>`_)
+  * add CI jobs for building and testing workspaces defines by a .repos file (`#590 <https://github.com/ros-infrastructure/ros_buildfarm/pull/590>`_, `#607 <https://github.com/ros-infrastructure/ros_buildfarm/pull/607>`_, `#610 <https://github.com/ros-infrastructure/ros_buildfarm/pull/610>`_, `#623 <https://github.com/ros-infrastructure/ros_buildfarm/pull/623>`_, `#628 <https://github.com/ros-infrastructure/ros_buildfarm/pull/628>`_, `#629 <https://github.com/ros-infrastructure/ros_buildfarm/pull/629>`_, `#630 <https://github.com/ros-infrastructure/ros_buildfarm/pull/630>`_, `#632 <https://github.com/ros-infrastructure/ros_buildfarm/pull/632>`_, `#633 <https://github.com/ros-infrastructure/ros_buildfarm/pull/633>`_, `#636 <https://github.com/ros-infrastructure/ros_buildfarm/pull/636>`_)
```
Comment (gavanderhoorn) [inline CHANGELOG.rst]:
```suggestion
  * add CI jobs for building and testing workspaces defined in a .repos file (`#590 <https://github.com/ros-infrastructure/ros_buildfarm/pull/590>`_, `#607 <https://github.com/ros-infrastructure/ros_buildfarm/pull/607>`_, `#610 <https://github.com/ros-infrastructure/ros_buildfarm/pull/610>`_, `#623 <https://github.com/ros-infrastructure/ros_buildfarm/pull/623>`_, `#628 <https://github.com/ros-infrastructure/ros_buildfarm/pull/628>`_, `#629 <https://github.com/ros-infrastructure/ros_buildfarm/pull/629>`_, `#630 <https://github.com/ros-infrastructure/ros_buildfarm/pull/630>`_, `#632 <https://github.com/ros-infrastructure/ros_buildfarm/pull/632>`_, `#633 <https://github.com/ros-infrastructure/ros_buildfarm/pull/633>`_, `#636 <https://github.com/ros-infrastructure/ros_buildfarm/pull/636>`_)
```

File: CHANGELOG.rst [hunk 2]
```diff
@@ -2,6 +2,50 @@
...
+  * add options to configure apt/pip package dependencies for the independent doc job in the build file (`#618 <https://github.com/ros-infrastructure/ros_buildfarm/pull/618>`_)
+  * [prerelease] add ability to generate repos files for faster cloning (rebased) (`#600 <https://github.com/ros-infrastructure/ros_buildfarm/pull/600>`_)
+  * only consider same type distros when looking for previous distro (`#593 <https://github.com/ros-infrastructure/ros_buildfarm/pull/593>`_)
+  * share ccache between docker builds (`#580 <https://github.com/ros-infrastructure/ros_buildfarm/pull/580>`_)
+  * allow searching by email on status pages (`#561 <https://github.com/ros-infrastructure/ros_buildfarm/pull/561>`_)
+  * set build environment variables from build files (`#554 <https://github.com/ros-infrastructure/ros_buildfarm/pull/554>`_, `#558 <https://github.com/ros-infrastructure/ros_buildfarm/pull/558>`_)
+  * add devel job test statistics collation (`#541 <https://github.com/ros-infrastructure/ros_buildfarm/pull/541>`_)
+
+* Changes
```
Comment (gavanderhoorn) [inline CHANGELOG.rst]:
```suggestion
* Changes

```

**Comment by dirk-thomas** [comment]
@gavanderhoorn Thanks for the fixes.

**Comment by gavanderhoorn** [comment]
@dirk-thomas: github's "latest release" still points to `2.0.0`, not `3.0.0` [here](https://github.com/ros-infrastructure/ros_buildfarm/releases). Is that intentional?

**Comment by dirk-thomas** [comment]
> github's "latest release" still points to 2.0.0, not 3.0.0 here. Is that intentional?

Absolutely not, thanks for the reminded. This is one of the few repos actually using the GitHub releases. I simply forgot about it. Done.

**Pull request merged**